widgets/Layout/withAuthLayout/WithAuthLayout.module.scss

```diff
@@ -0,0 +1,3 @@
+.WithAuthLayout {
+  margin-left: 220px;
```

Comment:
There is no `padding` for the content in the styles of this container.
The name `contentWrapper` would make more sense.